google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/project/ProjectSelectionDialog.java

```diff
@@ -122,17 +121,12 @@ CloudProject showDialog(@Nullable CloudProject cloudProject) {
       installTableSpeedSearch(projectListTable);
       installProgressBar(dialogWrapper);
 
-      Stream.of(ProjectManager.getInstance().getOpenProjects())
-          .forEach(
-              project ->
-                  project
-                      .getMessageBus()
-                      .connect(dialogWrapper.getDisposable() /* disconnect once dialog is gone. */)
-                      .subscribe(
-                          GoogleLoginListener.GOOGLE_LOGIN_LISTENER_TOPIC,
-                          () -> {
-                            refreshDialog(Services.getLoginService().getActiveUser());
-                          }));
+      ApplicationManager.getApplication()
```

Comment:
I guess its ok to use the Application wide bus because you are still passing in the disposable? I.e. it will always get cleaned when the dialog is closed.

Reply:
Yep, I think it's the same mechanics.